### ISSUE TITLE
Feature Request - MvxRecyclerView should support incremental loading

### ISSUE THREAD

**Issue by thefex** [MvvmCross/MvvmCross-AndroidSupport#205]
MvxRecyclerView should support incremental loading (fetching new items) by scrolling to the end of list.
I propose adding binding such as 
local:MvxBind="ItemSource SomeObservableItemsSource; IncrementalLoading CommandWhichHandleIncrementalLoading"

**Comment by martijn00** [comment]
There is already a plugin for that: https://github.com/HBSequence/Sequence.Plugins

**Comment by thefex** [comment]
@martijn00 Hm it doesn't support recyclerview but looks it will be better to fork plugin and add recyclerview support there.
Closing.

**Comment by Cheesebaron** [comment]
Why not add it upstream in the IMvxRecylerViewAdapter?

**Comment by martijn00** [comment]
We could do that, but it might be better to keep things centralized in one plugin.

**Comment by Cheesebaron** [comment]
I don't think it should be another plugin, because the plugin provides a View replacement and an Adapter replacement and isn't really something you plug-in...

**Comment by ilber** [comment]
Hi @Cheesebaron,
Thank you for your amazing job first.
I wanted to ask you if there is any update here with the continuous loading here?
Thanks again.
